src/core/linalg/src/sparse/4C_linalg_vector.hpp

```diff
@@ -75,232 +72,232 @@ namespace Core::LinAlg
+    int replace_global_value(int GlobalRow, int VectorIndex, double ScalarValue)
     {
       return vector_->ReplaceGlobalValue(GlobalRow, VectorIndex, ScalarValue);
     }
 
-    int ReplaceGlobalValue(long long GlobalRow, int VectorIndex, double ScalarValue)
+    int replace_global_value(long long GlobalRow, int VectorIndex, double ScalarValue)
     {
       return vector_->ReplaceGlobalValue(GlobalRow, VectorIndex, ScalarValue);
     }
 
     //! Matrix-Matrix multiplication, \e this = ScalarThis*\e this + ScalarAB*A*B.
-    int Multiply(char TransA, char TransB, double ScalarAB, const Epetra_MultiVector& A,
+    int multiply(char TransA, char TransB, double ScalarAB, const Epetra_MultiVector& A,
         const Epetra_MultiVector& B, double ScalarThis)
     {
       return vector_->Multiply(TransA, TransB, ScalarAB, A, B, ScalarThis);
     }
 
     //! Puts element-wise reciprocal values of input Multi-vector in target.
-    int Reciprocal(const Epetra_MultiVector& A) { return vector_->Reciprocal(A); }
+    int reciprocal(const Epetra_MultiVector& A) { return vector_->Reciprocal(A); }
 
     //! Multiply a Core::LinAlg::MultiVector<double> with another, element-by-element.
-    int Multiply(double ScalarAB, const Epetra_MultiVector& A, const Epetra_MultiVector& B,
+    int multiply(double ScalarAB, const Epetra_MultiVector& A, const Epetra_MultiVector& B,
         double ScalarThis)
     {
       return vector_->Multiply(ScalarAB, A, B, ScalarThis);
     }
 
-    int ReplaceGlobalValues(int NumEntries, const double* Values, const int* Indices)
+    int replace_global_values(int NumEntries, const double* Values, const int* Indices)
     {
       return vector_->ReplaceGlobalValues(NumEntries, Values, Indices);
     }
 
-    int ReplaceGlobalValues(int NumEntries, const double* Values, const long long* Indices)
+    int replace_global_values(int NumEntries, const double* Values, const long long* Indices)
     {
       return vector_->ReplaceGlobalValues(NumEntries, Values, Indices);
     }
 
     //! Imports an Epetra_DistObject using the Epetra_Import object.
-    int Import(const Epetra_SrcDistObject& A, const Epetra_Import& Importer,
+    int import(const Epetra_SrcDistObject& A, const Epetra_Import& Importer,
         Epetra_CombineMode CombineMode, const Epetra_OffsetIndex* Indexor = nullptr)
     {
       return vector_->Import(A, Importer, CombineMode, Indexor);
     }
 
     //! Imports an Epetra_DistObject using the Epetra_Export object.
-    int Import(const Epetra_SrcDistObject& A, const Epetra_Export& Exporter,
+    int import(const Epetra_SrcDistObject& A, const Epetra_Export& Exporter,
         Epetra_CombineMode CombineMode, const Epetra_OffsetIndex* Indexor = nullptr)
     {
       return vector_->Import(A, Exporter, CombineMode, Indexor);
     }
 
-    int Export(const Epetra_SrcDistObject& A, const Epetra_Import& Importer,
+    int export_to_dist_object(const Epetra_SrcDistObject& A, const Epetra_Import& Importer,
```

Comment:
Minor: Maybe just `export_to`? I also used this in other places where `export` is not possible.